.github/workflows/build-rock.yaml

```diff
@@ -39,7 +39,7 @@ jobs:
           path: '*.rock'
 
   spread-tests:
-    runs-on: self-hosted
+    runs-on: [self-hosted, spread-installed]
```

Comment:
I don't think there are runners with this tag available for this repo.  This was pending indefinitely when I tried it and it looks like it's pending for you too.

I don't have admin so I can't see which runners are available.  Snapcraft has specific runners available with `spread-installed` that aren't part of the standard Canonical runner pool.

Reply:
Yeah that's a bit weird. I didn't think our standard self-hosted runners had spread installed. It looks like the older ones are running on [a correct runner](https://github.com/canonical/snapcraft-rocks/actions/runs/15428034587/job/43420212993#step:1:2), but I think that's mostly luck that it hasn't happened to run on one of the shared runners before.

Can someone with admin access to this repo (@sergiusens or @tigarmo I think?) check what tags are on the spread runners? For Charmcraft they look like this:

![image](https://github.com/user-attachments/assets/51b5cf7e-4504-432d-9ff1-ba9fb0639f75)

Reply:
Looks like @tigarmo got this fixed (thanks!), but I don't think the [project prepare failure](https://github.com/canonical/snapcraft-rocks/actions/runs/15570925017/job/43903429506?pr=72) on 20.04 is related to this PR?

Reply:
Yes please try installing with apt ([here](https://github.com/canonical/snapcraft-rocks/blob/core24-8/spread.yaml#L70)), although there's a fair chance that focal's docker is too old

Reply:
I'm going to put that in a separate PR. I'll keep this open until the other one lands.